### PR TITLE
add advisory for actix-http HRS

### PR DESCRIPTION
**Pull request by robjtede**

### CODE REVIEW

File: crates/actix-http/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "actix-http"
+date = "2021-06-16"
+categories = ["web-security"]
+keywords = ["smuggling", "http", "reverse proxy", "request smuggling"]
+
+[versions]
+patched = [">= 2.2.1", ">= 3.0.0-beta.9"]
```
Comment (Shnatsel) [inline crates/actix-http/RUSTSEC-0000-0000.md:10]:
```suggestion
patched = ["^ 2.2.1", ">= 3.0.0-beta.9"]
```

The version specification with two `>=` would be redundant: `3.0.0-beta.9` is greater than `2.2.1` under [semver precedence rules](https://semver.org/#spec-item-11). I assume you mean to specify that everything in 2.x series after `2.2.1` and also anything greater than `3.0.0-beta.9` ever?

Comment (robjtede) [reply crates/actix-http/RUSTSEC-0000-0000.md:10]: Yes that's true. Copy pasted the other actix http advisory and forgot to change the semver prefix.

**Comment by Shnatsel** [comment]
I've adjusted the advisory to make CI pass and merged it. Thanks for the report!

**Comment by desandy** [comment]
Is there going to be a CVE for this issue?

**Comment by robjtede** [comment]
[CVE-2021-38512](https://nvd.nist.gov/vuln/detail/CVE-2021-38512)

**Pull request merged**